src/openwrt/ubus/otubus.cpp

```diff
@@ -1103,16 +1103,20 @@ int UbusServer::UbusGetInformation(struct ubus_context *     aContext,
     }
     else if (!strcmp(aAction, "networkkey"))
     {
-        char           outputKey[NETWORKKEY_LENGTH] = "";
-        const uint8_t *key = reinterpret_cast<const uint8_t *>(otThreadGetNetworkKey(mController->GetInstance()));
-        OutputBytes(key, OT_NETWORK_KEY_SIZE, outputKey);
+        char         outputKey[NETWORKKEY_LENGTH] = "";
+        otNetworkKey key;
+
+        otThreadGetNetworkKey(mController->GetInstance(), &key);
+        OutputBytes(key.m8, OT_NETWORK_KEY_SIZE, outputKey);
         blobmsg_add_string(&mBuf, "Networkkey", outputKey);
     }
     else if (!strcmp(aAction, "pskc"))
     {
-        char          outputPskc[NETWORKKEY_LENGTH] = "";
-        const otPskc *pskc                          = otThreadGetPskc(mController->GetInstance());
-        OutputBytes(pskc->m8, OT_NETWORK_KEY_SIZE, outputPskc);
+        char   outputPskc[NETWORKKEY_LENGTH] = "";
+        otPskc pskc;
+
+        otThreadGetPskc(mController->GetInstance(), &pskc);
+        OutputBytes(pskc.m8, OT_NETWORK_KEY_SIZE, outputPskc);
```

Comment:
OT_NETWORK_KEY_SIZE and OT_PSKC_MAX_SIZE are both 16, but we'd better use OT_PSKC_MAX_SIZE for better readability. We can enhance later.

Reply:
@bukepo I have fixed this.